### ISSUE TITLE
Decreased acid sphingomyelinase activity 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7393]
**Preferred term label:**
Decreased acid sphingomyelinase activity

**Synonyms**

**Definition (free text, please give PubMed ID)**
Reduced enzyme  of  acid sphingomyelinase activity in the blood circulation.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
607616 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
